src/driver/driver_api.cc

```diff
@@ -155,6 +156,13 @@ transform::Pass BindTarget(Target target) {
   return tir::transform::CreatePrimFuncPass(fpass, 0, "BindTarget", {});
 }
 
+transform::Pass AnnotateEntryFunc(bool b) {
```

Comment:
should this be static as it's mainly helper, or do you envision others calling it from outside this module?